tensorflow/python/ops/nn_impl.py

```diff
@@ -504,26 +504,27 @@ def _swish_shape(op):
   return [op.inputs[0].shape]
 
 
-@function.Defun(shape_func=_swish_shape, func_name="swish_grad", noinline=True)
+@function.Defun(shape_func=_swish_shape, func_name="swish_grad")
```

Comment:
The right answer here is to remove function.Defun altogether and replace its usage of a custom gradient with custom_gradient.custom_gradient.

Reply:
Done. Let me know if it looks OK.